addon/model-data.js

```diff
@@ -88,13 +113,20 @@ export default class M3ModelData {
     }
 
     if (notifyRecord) {
```

Comment:
I'm going to refactor this a little to be after `notifyProjectionProperties`, because otherwise we will have double notification for nested models (the tests for them are not enabled yet).